text/103-leaving-x-twitter.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: Transitioning from X/Twitter to new social media platforms
+Drivers: Théo Zimmermann (@Zimmi48)
+---
+
+## Summary
+
+This RFC proposes that as part of the actions to be taken in the context of the renaming of Coq to the Rocq Prover, a departure from X/Twitter should be announced and implemented. The rationale for this decision is based on the changing ownership and policy at X/Twitter, the current management of the X/Twitter account, the overall context of many academics and academic institutions leaving X/Twitter, and the opportunity presented by the renaming to move to more appropriate platforms.
+
+## Context and motivation
+
+The decision to leave X/Twitter is motivated by several factors:
+
+1. **Changing ownership and policy at X/Twitter:**
+  Since the acquisition of Twitter by Elon Musk, significant policy changes, including in moderation, promotion of misinformation, etc. have caused concern within the scientific and open-source communities. The Coq/Rocq team's values of openness, collaboration, and inclusivity, but also the academic and scientific communities' values, are increasingly at odds with the direction of the platform.
+
+1. **Costs associated with the renaming:**
+  Renaming Coq to the Rocq Prover triggers costs, including for the X/Twitter account, which will also need to be renamed. Doing the work of adapting the current CoqLang account to become a Rocq Prover X/Twitter account in the current context would send a political statement that the project does not care about the current direction the platform is taking. Disabling the existing account and stopping all use of X/Twitter is the only reasonable course of action in the context of the renaming.
+
+1. **Diminishing engagement and influence:**
+  [X/Twitter's relevance among academics is waning](https://www.insidehighered.com/news/tech-innovation/digital-publishing/2024/11/13/more-academics-flee-x-after-election), with many users and academic institutions (e.g., Aix-Marseille University, University of Strasbourg, University of Luxembourg, RWTH Aachen, Utrecht University, York University) having already left, and migrating to alternatives such as Mastodon and Bluesky. Furthermore, X/Twitter is already banned in Brazil, and an investigation against the platform by the EU commission is ongoing. In this context, staying active on X/Twitter is no longer a good use of community resources.
```

Comment:
> Furthermore, X/Twitter is already banned in Brazil

Per [Wikipedia](https://en.wikipedia.org/wiki/Blocking_of_Twitter_in_Brazil), the ban was rescinded on Oct 8, 2024.

> an investigation against the platform by the EU commission is ongoing

How do we know whether there's merit to the investigation or it's just a pressure tactic?

Reply:
> > Furthermore, X/Twitter is already banned in Brazil
> 
> Per [Wikipedia](https://en.wikipedia.org/wiki/Blocking_of_Twitter_in_Brazil), the ban was rescinded on Oct 8, 2024.

Good to know. Yet, this is a perfect example of the problematic behavior of the new X/Twitter management, which needs to lead to extreme retaliation.

> > an investigation against the platform by the EU commission is ongoing
> 
> How do we know whether there's merit to the investigation or it's just a pressure tactic?

Pressure tactic for what? Respecting the EU law?